src/ringmesh/basic/plugin_manager.cpp

```diff
@@ -128,6 +188,30 @@ namespace RINGMesh
             }
         }
 
+        std::string home_directory() const
+        {
+            std::string homedir{ getenv( "HOME" ) };
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Replace this call to the non reentrant function 'getpwuid' by a call to 'getpwuid_r'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3ANonReentrantFunction)